### PR TITLE
HBASE-23002 [HBCK2/hbase-operator-tools] Create an assembly that buil…

### PR DESCRIPTION
**Pull request by saintstack**
…ds an hbase-operator-tools tgz

Add an hbase-assembly that is just for creating assemblies.
Also changed groupid from org.apache.hbase to
org.apache.hbase.operator.tools so lands into a different area in
repository -- less crowded, easier to differentiate from core
hbase.

Creates a bin assembly (src is made elsewhere in release script).
Adds a first-cut at RELEASENOTES and CHANGES (release script will
fix up what is here based off what is in JIRA to make it suit
the 1.0.0RC0)

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# HBASE Operator Tools Changelog
```
Comment (petersomogyi) [inline CHANGES.md]: Why the CHANGES file included with the assembly module's introduction?

File: RELEASENOTES.md [hunk 2]
```diff
@@ -0,0 +1,1228 @@
+# RELEASENOTES
```
Comment (petersomogyi) [inline RELEASENOTES.md]: Same with RELEASENOTES

File: hbase-operator-tools-assembly/src/main/assembly/bin.xml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    <file>
+      <source>${project.parent.basedir}/CHANGES.md</source>
+      <outputDirectory>.</outputDirectory>
+      <destName>CHANGES.md</destName>
+      <lineEnding>unix</lineEnding>
+    </file>
+    <file>
+      <source>${project.parent.basedir}/README.md</source>
+      <outputDirectory>.</outputDirectory>
+      <destName>README.md</destName>
```
Comment (petersomogyi) [inline hbase-operator-tools-assembly/src/main/assembly/bin.xml]: README.md is also included in `moduleSets`. Are both of these lines are required?

File: CHANGES.md (hunk 1 above)
Comment (saintstack) [reply CHANGES.md]: We want CHANGES and RELEASENOTES included in the binary tgz, right? Need placeholders.

File: RELEASENOTES.md (hunk 2 above)
Comment (saintstack) [reply RELEASENOTES.md]: Ditto

File: hbase-operator-tools-assembly/src/main/assembly/bin.xml (hunk 3 above)
Comment (saintstack) [reply hbase-operator-tools-assembly/src/main/assembly/bin.xml]: One is the high-level README for the repo. The other is for hbase-hbck.

**Comment by saintstack** [comment]
Thanks for review Peter.

**Comment by saintstack** [comment]
>  I assume the content of changes and releasenotes will be updated on release creation time.

Script fills in Release notes and Changes... yes sir. Thanks for review.
